README.md

```diff
@@ -26,15 +26,15 @@ integrate Apruve.
 
 ### Create an account on Apruve.
 
-Have a look though our [Getting Started documentation](https://www.apruve.com/doc/developers/) and be sure to use
+Have a look though our [Getting Started documentation](https://docs.apruve.com/guides/getting-started-f09a17ef-053e-4eba-9141-6fe8ce3b774d) and be sure to use
```

Comment:
As far as I can tell, Gelato gives us no control over that.  That's just the link they generated for that guide.  Maybe because of the generic title (Getting Started)?